example/spec/parallel/kinesisTests/KinesisReplaySpec.js

```diff
@@ -215,6 +216,10 @@ describe('The Kinesis Replay API', () => {
         console.log('Waiting to ensure workflows expected not to start do not start...');
         await Promise.all(tooOldToExpectWorkflows);
         await Promise.all(tooNewToExpectWorkflows);
+
+        console.log('Cleaning up executions...');
+        await deleteExecution({ prefix: testConfig.stackName, executionArn: workflowExecutions[0].workflowArn });
+        await deleteExecution({ prefix: testConfig.stackName, executionArn: workflowExecutions[1].workflowArn });
```

Comment:
Consider  a promise.allSettled / catch.   Possibly a DRY'd up helper function for these sorts of cleanups?

Reply:
Yup! These don't need to be in order. Also, I think this is one of the last errors I was looking for. I was using the wrong property here and it was throwing a 404. Updated here:

https://github.com/nasa/cumulus/pull/2252/commits/8106aed787b4b836b09d21cd6af8dca6f3a9e4d3